### ISSUE TITLE
Tree Explorer hides at every start

### ISSUE THREAD

**Issue by steo82rm** [giuspen/cherrytree#2126]
The Tree Explorer hides every time I close and I open again the CherryTree program.
Currently it is set to be visible on the right side but sometimes even when it is on the left side it presents the same behavior.

The Tree Explorer doesn't retain the last position/size as it was set by the user.

**Comment by giuspen** [comment]
What is the cherrytree version and on what operative system

**Comment by steo82rm** [comment]
cherrytree v.0.99.47-2 on Linux (Manjaro) 
thank you

**Comment by giuspen** [comment]
Can you test the current version 0.99.49? You can use the AppImage (single portable executable file) if there is not a package for your distribution see https://www.giuspen.net/cherrytree/#downl

**Comment by steo82rm** [comment]
Just updated the whole system and also the new CherryTree version has been installed has required.
Apparently the same behavior is still there and happens every time.
Note: not sure if this can be useful, but if I move the Tree Explorer side panel on the left side of the screen (from the CherryTree settings) the problem than doesn't happen, but of course moving it back to the right side will make it happen again.
Basically it doesn't completely disappear but it shrink to the right side of the screen and doesn't retain the size set by the user.
Thank you/Grazie!

**Comment by steo82rm** [comment]
After new updates problem still persist.
Looking at the configuration file I see this but I am not sure if there is any parameter that may cause the reported issue:

```
[tree]
rest_exp_coll=0
nodes_bookm_exp=false
nodes_icons=b
aux_icon_hide=false
default_icon_text=25
tree_right_side=true
cherry_wrap_ena=false
cherry_wrap_width=70
tree_click_focus_text=false
tree_click_expand=false
```

Thank you again for the attention to this.

**Comment by steo82rm** [comment]
Update!
Ok, after a few tests (even removing the configuration file) I have discovered that CherryTree will keep the tree visible as I left before quitting it if the windows is not maximized to full screen. If I close it NOT maximized at the next execution the program will start full screen (I mean maximized) and with the tree visible exactly as I left it. Also, if I then close it maximized then it will present again the same issue at the next execution.
I hope this can address better the possible issue.
Thank you!

**Comment by giuspen** [comment]
Sure your further tests help narrow down, I'll have a go and see if I can reproduce

**Comment by giuspen** [comment]
I was able to reproduce, will be fixed in next release 0.99.51
